Library/Sender.ts

```diff
@@ -146,6 +165,8 @@ class Sender {
                             }
                             // store to disk in case of burst throttling
                         } else if (
+                            res.statusCode === 401 || // Unauthorized
+                            res.statusCode === 403 || // Forbidden
```

Comment:
So there is no retry when we hit any AAD auth error?  Is that what other langs are doing?  I thought we were going to retry!

Reply:
This specific code is the retry logic, envelopes would be put into disk, are you expecting something else?

Reply:
Yes, just looked at Python PR.  Leighton is retrying if we hit 401. @lzchen can you comment?

Reply:
Yes 401 is a retry scenario.